### PR TITLE
add recipe for unifrac

### PR DESCRIPTION
**Pull request by gwarmstrong**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ElDeveloper** [comment]
I'm willing to be listed as a maintainer of this feedstock.

**Comment by gwarmstrong** [comment]
@conda-forge/help-python-c I think we are ready for a review here

**Comment by ElDeveloper** [comment]
@conda-forge/help-python-c, I think we are ready for review here.

Not sure the message above worked (since the markup does not display the tag as a hyperlink).

File: recipes/unifrac/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  url: https://github.com/biocore/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 97507caa5536bf445ca712ec495a39c4c95db3388d3259be3270e0e15ceb5ce5 
+
+build:
+  number: 0
+  skip: True  # [py<35 or win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
```
Comment (chrisburr) [inline recipes/unifrac/meta.yaml]: This should be the only `build` dependency, the rest should be in `host`.

File: recipes/unifrac/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - hdf5 >=1.8.17
+    - h5py
+    - biom-format
+  host:
+    - python
+    - pip
+    - numpy >=1.12.1
+    - Cython >=0.28.3
+  run:
+    - scikit-bio >=0.5.1
```
Comment (chrisburr) [inline recipes/unifrac/meta.yaml]: I presume you need Python at run time?
```suggestion
    - python
    - scikit-bio >=0.5.1
```

File: recipes/unifrac/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    - biom-format
+    - numpy >=1.12.1
```
Comment (chrisburr) [inline recipes/unifrac/meta.yaml]: See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=numpy#building-against-numpy
```suggestion
    - {{ pin_compatible('numpy') }}
```

File: recipes/unifrac/LICENSE [hunk 4]
```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```
Comment (chrisburr) [inline recipes/unifrac/LICENSE]: There is no need for this file as it is included in the tarball

File: recipes/unifrac/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+build:
+  number: 0
+  skip: True  # [py<35 or win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - biom-format
+    - Cython >=0.28.3
```
Comment (scopatz) [inline recipes/unifrac/meta.yaml]: package names are all lowercase

**Comment by gwarmstrong** [comment]
@chrisburr @scopatz I have addressed the feedback here

**Comment by mbargull** [comment]
This package seems bio-related and is maintained on Bioconda since a couple of months (>10k downloads, cumulatively):
https://anaconda.org/bioconda/unifrac/files
https://github.com/bioconda/bioconda-recipes/blob/9744655d24894a303a164154fa2a893d82330fdb/recipes/unifrac/meta.yaml

Do we have good reasons to move it to conda-forge?
(pinging recipe contributors from Bioconda: @ebolyen, @npavlovikj, @bgruening)

**Comment by npavlovikj** [comment]
@mbargull , thank you for spotting this. I am not user of the package, but I do think the recipe is bio-related (performs phylogenetic calculations) and is more suitable for `bioconda`. Looks like the `bioconda` recipe was updated to 0.10.0 ~20 hours ago (after this PR), so maybe updating the version is the reason for this PR?

**Comment by gwarmstrong** [comment]
After some discussions, we agree that this belongs on bioconda. Thanks @everyone for your time!

**Comment by mbargull** [comment]
> Thanks everyone for your time!

Seconded! Thanks for the reviews anyway.
@gwarmstrong, @ElDeveloper, if there are important differences between the recipe on Bioconda and the one you worked on here, please feel free to suggest those changes on https://github.com/bioconda/bioconda-recipes/ . In any case, thanks for your contributions, no matter if here or @ Bioconda!

**Pull request closed without merging**